### PR TITLE
fix: graph iteration in case it has no edges

### PR DESCRIPTION
**Pull request by davidbp**
This PR Fixes iteration over graphs with no edges

### CODE REVIEW

**Review by JoanFM — COMMENTED** [review]
Please add a test that shows it does not crash

**Review by JoanFM — CHANGES_REQUESTED** [review]
tests to be added

**Comment by davidbp** [comment]
> tests to be added

added a simple test to verify iteration is done without breaking on grah without edges

**Review by JoanFM — CHANGES_REQUESTED** [review]
Add test to create `DGLGraph` from graph with no edges

File: tests/unit/types/document/test_graph_document.py [hunk 1]
```diff
@@ -178,6 +178,13 @@ def test_from_dgl_graph(graph):
     assert (graph.adjacency.col == jina_graph.adjacency.col).all()
     assert (graph.adjacency.col == jina_graph.adjacency.col).all()
 
+    from dgl import DGLGraph
```
Comment (JoanFM) [inline tests/unit/types/document/test_graph_document.py:181]: can this be an individual test with a meaningful test (test_convert_no_edges_graph) or something?

**Review by JoanFM — APPROVED** [review]

**Pull request merged**